### PR TITLE
Add go-perl

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
@bioconda/core please review

File: recipes/perl-go-perl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+{% set version = "0.15" %}
+{% set sha256 = "423d26155ee85ca51ab2270cee59f4e85b193e57ac3a29aff827298c0a396b12" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: go-perl-0.15.tar.gz
+  url: https://cpan.metacpan.org/authors/id/C/CM/CMUNGALL/go-perl-0.15.tar.gz
```
Comment (bgruening) [inline recipes/perl-go-perl/meta.yaml]: Please use the variable for the version here as well.

File: recipes/perl-go-perl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+test:
+  imports:
+    - GO::Parser
+  commands:
+    - which map2slim >& /dev/null
+
+about:
+  home: http://metacpan.org/pod/go-perl
+  license: Perl
```
Comment (bgruening) [inline recipes/perl-go-perl/meta.yaml]: I could not find a real license, which is a problem, do you have more luck?

File: recipes/perl-go-perl/meta.yaml (hunk 1 above)
Comment (sebastian-luna-valero) [reply recipes/perl-go-perl/meta.yaml]: Done

File: recipes/perl-go-perl/meta.yaml (hunk 2 above)
Comment (sebastian-luna-valero) [reply recipes/perl-go-perl/meta.yaml]: If you look at http://metacpan.org/pod/go-perl it says `unknown` (?)

**Comment by sebastian-luna-valero** [comment]
Please note that this recipe was created with `conda skeleton cpan go-perl`

**Comment by sebastian-luna-valero** [comment]
@bgruening can I please go ahead without a license?

File: recipes/perl-go-perl/meta.yaml (hunk 2 above)
Comment (johanneskoester) [reply recipes/perl-go-perl/meta.yaml]: I think we should say unknown then.

**Comment by sebastian-luna-valero** [comment]
Thanks both. Will merge but please let me know any other concerns you may have.

**Comment by bgruening** [comment]
@sebastian-luna-valero the big problem with no-license is that strictly speaking we are not allowed to distribute it. Maybe you can contact upstream and ask to fix this?

**Comment by sebastian-luna-valero** [comment]
@bgruening the author came back to me so I am updating the recipe in https://github.com/bioconda/bioconda-recipes/pull/7062

**Comment by bgruening** [comment]
Awesome thanks!

**Pull request merged**